### ISSUE TITLE
Add-on config 'devices' use a deprecated format

### ISSUE THREAD

**Issue by Veldkornet** [hassio-addons/addon-ssh#196]
# Problem/Motivation

> Warning appears in the supervisor log

## Expected behavior

> No warnings from the add-on

## Actual behavior
```
21-02-02 16:54:13 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of SSH & Web Terminal
```
## Steps to reproduce

> Use add-on 8.0.0 with HA 2021.2.0b2

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by Veldkornet** [comment]
This appears for all of the add-ons FYI, Visual Sudio code, Zwavejs2mqtt, ESPHome, deconz, etc

**Comment by frenck** [comment]
Yes, and we cannot migrate yet, as it is only available in beta

**Comment by frenck** [comment]
Fixed in #202
